src/Schedule.h

```diff
@@ -373,6 +445,24 @@ class StageSchedule {
     std::vector<PrefetchDirective> &prefetches();
     // @}
 
+    /** Until which loop level (starting from outermost) we should fuse
```

Comment:
"Innermost loop level of fused loop nest for this function stage. Fusion runs from outermost to this loop level." (Definitely remove question mark unless I'm missing something.)